impl/events.go

```diff
@@ -152,6 +152,13 @@ func (m *manager) OnResponseReceived(chid datatransfer.ChannelID, response datat
 	return m.resumeOther(chid)
 }
 
+func (m *manager) OnRequestTimedOut(chid datatransfer.ChannelID) error {
+	// TODO Start a timer to cleanup state if this dosen't complete in a while
```

Comment:
@hannahhoward Have created  issue https://github.com/filecoin-project/lotus/issues/3878 to clean these up from the FSM.